doc/build.jl

```diff
@@ -163,14 +163,14 @@ Centrality measures implemented in *LightGraphs.jl* include the following:
 *LightGraphs.jl* implements some common random graph generators:
 
 {{erdos_renyi, watts_strogatz, random_regular_graph, random_regular_digraph,
-    random_configuration_model, stochastic_block_model,
+    random_configuration_model, barabasi_albert, stochastic_block_model,
```

Comment:
yes, I'm sorry, couldn't resist. This is a recently added function with no correspondence in the docs